lib/tasks/gettext.rake

```diff
@@ -20,10 +20,13 @@ begin
     end
   end
 
-  desc 'Extract plugin strings - called via rake plugin:gettext[plugin_name]'
-  task 'plugin:gettext', :engine do |t, args|
-    @domain = args[:engine]
-    @engine = "#{@domain.camelize}::Engine".constantize
+  desc 'Extract plugin strings - called via rake plugin:gettext[gettext_domain]'
```

Comment:
I was thinking about making this more like what we have for plugin:apipie:cache:
https://github.com/theforeman/foreman/blob/96c6d188e0d3792607f1de0905ee2de2ca33d2b4/lib/tasks/plugin_apipie.rake#L1-L9

Though it should be noted that `plugin:assets:precompile` takes a plugin name:
https://github.com/theforeman/foreman/blob/96c6d188e0d3792607f1de0905ee2de2ca33d2b4/lib/tasks/plugin_assets.rake#L1-L2

If only all `plugin:*` tasks were consistent ...

Reply:
Made the rake task accept plugin name